src/components/container/HeartEssentialEight/HeartEssentialEight.tsx

```diff
@@ -0,0 +1,215 @@
+      if (vv === didAchieve) {
+        const title = key.replace(/([A-Z])/g, " $1").replace(/^./, (str) => {
+          return str.toUpperCase();
+        });
+        let icon: any;
+        let subtitle: any;
+        let url: string;
+        const circleClasses: string[] = ["circle"];
+        switch (key) {
+          case "Activity":
+            icon = iconGetActive;
+            subtitle="Display Active Time from Fitbit / Garmin.  Probably AppleExerciseTime for Apple Health. Or manual entry";
+            circleClasses.push("activity");
+            break;
+          case "BloodPressure":
+            icon = iconManageBloodPressure;
+            subtitle="Display most recent value from EHR, or manual entry";
+            circleClasses.push("bloodPressure");
+            break;
+          case "BloodSugar":
+            icon = iconReduceBloodSugar;
+            subtitle="Display Glucose numbers from Dexcom, or manual entry";
+            circleClasses.push("bloodSugar");
+            break;
+          case "BMI":
+            icon = iconLoseWeight;
+            subtitle="Display BMI calculated from survey data"
+            circleClasses.push("bmi");
+            break;
+          case "Cholesterol":
+            icon = iconCholesterol;
+            subtitle="Display most recent value from EHR, or manual entry";
+            circleClasses.push("cholesterol");
+            break;
+          case "Diet":
+            icon = iconEatWell;
+            subtitle="Eat more whole foods / Cut back on candy / Whatever the key points are from the survey";
+            circleClasses.push("diet");
+            break;
+          case "Nicotine":
+            icon = iconStopSmoking;
+            subtitle="Stop smoking / Congratulations on { 5+ years/never smoking }"
+            circleClasses.push("nicotine");
+            break;
+          case "Sleep":
+            icon = iconSleep;
+            subtitle="Display value from Apple Health/Fitbit/Garmin, or manual entry"
+            circleClasses.push("sleep");
+            break;
+          default:
+            icon = null;
+        }
+
+        const achievementIcon = (
+          <div className={circleClasses.join(" ")}>
+            <img src={icon} />
+          </div>
+        );
+
+        const achievement = (
+            <Action
+              key={`achievement-${key}`}
+              title={title}
+              subtitle={subtitle}
+              icon={achievementIcon}
+              onClick={() => MyDataHelps.openExternalUrl(url)}
+            />
+        );
+        achievementList.push(achievement);
+      }
+    }
+
+    return achievementList;
+  }
+
+  useInitializeView(initialize, [], [props.previewState]);
+
+  return (
+    <>
+      {score === undefined && <LoadingIndicator />}
+      {score !== undefined && (
+      <>
+        <Card>
+          <div className="mdhui-heart-e8-score-container">
+            <h3 className="le8-header">Heart Health Score</h3>
```

Comment:
I think you can use `<Title order={3}>Heart Health Score</Title>` for these, perhaps with a custom margin, if that remains necessary.